### ISSUE TITLE
Searching while on page x opens page x instead of page 1 of results

### ISSUE THREAD

**Issue by vertigo220** [chocolatey/choco#1571]
If you perform a search, then navigate to another page, then do another search, it will put you on the results page for the new search that you were on before the search. For example, if you search for "browser" and go to page 2, then search for "firefox," you'll get page 2 of the results for the firefox search. It should always start you off with the first page.

**Comment by ferventcoder** [comment]
@vertigo220 this is not an issue with choco.exe, right? https://github.com/chocolatey/chocolatey.org/issues would be where to file this.

**Comment by ferventcoder** [comment]
Also, it's not considered good etiquette to file 10 issues at once.

**Comment by vertigo220** [comment]
Sorry, I'm not trying to be rude. I used it for a while and took notes, then when I was ready I came here to report my findings. It doesn't really make sense to wait a day or two (or whatever the appropriate amount of time is?) in between making a new issue just for the sake of it. I'm still fairly new to reporting issues on GitHub, so I don't know all the nuances; I'm just trying to be helpful by providing feedback. I didn't think that taking my time to try and help by doing so would be considered rude. Again, my apologies.
